.github/workflows/stale-issue-cleanup.yml

```diff
@@ -0,0 +1,18 @@
+name: 'Close stale issues and PRs'
+# Marks issues and PRs as stale after 30 days, then closes them if marked stale for 5 days
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v7
+        with:
+          stale-issue-message: 'This issue is stale because it has been open 30 days with no activity. Remove stale label or comment or this will be closed in 5 days.'
```

Comment:
suggest: Remove stale label or comment, **else** this will be closed in 5 days.

Reply:
same in `stale-pr-message`

Reply:
Updated